### DISCUSSION TITLE
Mutable Instruments

### DISCUSSION THREAD

**Discussion by clmpoirier** [FundamentalFrequency/.github#78] [Ideas]
Hi there,

I wondered if the LMN-3 could benefit from the Mutable Instruments modules?
[https://github.com/pichenettes](https://github.com/pichenettes)
See for instance a Teensy port of the Braids oscillator [https://github.com/modlfo/teensy-braids](https://github.com/modlfo/teensy-braids)

Thanks for your feedback!

**Comment by stonepreston** [comment]
Its certainly possible, but these would have to be ported to JUCE synths, which would be quite the task :P
